.env

```diff
@@ -0,0 +1 @@
+export URL=https://hub.dev.trade-tariff.service.gov.uk/
```

Comment:
more of a thought but do you think we will ever want a staging url?

Reply:
I figured we'd run these tests after a deploy to each environment, yeah. It should enable continuous deployment